src/css/components/help-menu.less

```diff
@@ -0,0 +1,34 @@
+/* =============
+ * Help Menu
+ * =============
+ */
+.help-menu {
+  float: left;
+  margin: 19px;
+  cursor: pointer;
+  color: transparent;
+  background-position: center left;
+  &.dropdown.popover.bottom {
```

Comment:
We should consider implementing a more abstract popover/overla/something component, as this kinda looks strange... Not part as of this PR. :)